fastlane/Fastfile

```diff
@@ -0,0 +1,35 @@
+system("npm install shelljs@0.7.0")
+system("npm install -g cordova")
+system("cordova telemetry on")
+
+$os = "ios,android"
+$jenkins = ENV["JENKINS_URL"]
+unless $jenkins.nil?
+	if $jenkins.include?"Android-Package"
+		$os = "android"
+  elsif $jenkins.include?"iOS-Package"
+		$os = "ios"
+	end
+end
+
+lane :all do
+	system("set -o pipefail && node ../test/test_force.js --os=#$os --apptype=native,native_swift,react_native,hybrid_remote,hybrid_local")
+	system("set -o pipefail && node test_force.js --os=#$os --templaterepouri=https://github.com/forcedotcom/SmartSyncExplorerReactNative#master")
```

Comment:
We should probably have a task in the release checklist to make sure all the pointers / tags are correct across all repos.